src/openassetio-python/cmodule/src/_openassetio.cpp

```diff
@@ -3,6 +3,14 @@
 
 #include "_openassetio.hpp"
 
+#ifdef OPENASSETIO_ENABLE_TESTS
+// In order to have representative tests of internal functionality, we
+// must unfortunately pollute this extension module with test-specific
+// bindings. The static library providing the implementation of this
```

Comment:
[consider][minor] Seems harsh. It's only conditionally compiled in and it serves a just purpose, and you dub it pollution? You are a cruel ruler indeed.

Reply:
I made it a bit less harsh. I took the liberty of squashing the commits.  The updated text can be found in 7e30300e.